### PR TITLE
Deprecate pl-threejs element

### PR DESCRIPTION
**Pull request by jonatanschroeder**
As discussed in a meeting. The corresponding library is outdated and use is not extensive enough to provide a stable testing background. The element continues to exist without change, but the docs no longer recommend it and the example course no longer includes an example for it.

### CODE REVIEW

**Review by eliotwrobson — APPROVED** [review]
Seems reasonable

File: exampleCourse/questions/element/threeJS/info.json
Comment (nwalters512) [inline exampleCourse/questions/element/threeJS/info.json:1]: I would rather move this question to `testCourse`. If we say we'll continue supporting it for backwards compatibility, we should ensure we still have a way to test it.

**Review by nwalters512 — APPROVED** [review]

File: exampleCourse/questions/element/threeJS/info.json
Comment (jonatanschroeder) [reply exampleCourse/questions/element/threeJS/info.json:1]: Done in 8412db7. I have not added it to any assessments, and have not restored the docs link to the question. Do you want me to keep the link in docs?

Comment (nwalters512) [reply exampleCourse/questions/element/threeJS/info.json:1]: No, this is really just for our internal use if we ever have to change anything with this element in the future.

**Pull request merged**